rpc_server.go

```diff
@@ -595,10 +597,8 @@ func RunRPCServer(portrpc int, block bool) {
 	var lsc LanceroSourceConfig
 	err = viper.UnmarshalKey("lancero", &lsc)
 	if err == nil {
-		err0 := sourceControl.ConfigureLanceroSource(&lsc, &okay)
-		if err0 != nil {
-			panic(err0)
-		}
+		sourceControl.ConfigureLanceroSource(&lsc, &okay)
+		// Don't panic on config errors: they are expected on a system w/o Lancero cards!
```

Comment:
```suggestion
if err0 := nil {
    log.Println("Error during ConfigureLanceroSource:", err)
```

Reply:
Not sure what you're proposing here, but the if-test will never succeed. Are you trying to get a log message? I'd put that in the `ConfigureLanceroSource` if you want that.

Reply:
Mostly I just don't want to have an unchecked error, we should check it, and do SOMETHING so I don't later come through and add a `panic` which is my default response to an unchecked error.

Reply:
Fair enough. I'll make it clearer. Probably the smart thing would be to not try to configure Lancero sources when no such device exists.